packages/cli/generators/model/templates/model.ts.ejs

```diff
@@ -1,7 +1,12 @@
-import {Entity, model, property} from '@loopback/repository';
+<% if (isModelBaseBuiltin) { -%>
+import {<%= modelBaseClass %>, model, property} from '@loopback/repository';
+<% } else { -%>
+import {model, property} from '@loopback/repository';
+import {<%= modelBaseClass %>} from '.';
```

Comment:
There actually is a good reason for importing from the model file instead of the index file.

Think about cyclic dependencies. For example, if `Customer` is extending `User` model, and the index file has re-exports sorted alphabetically (`Customer` first, `User` second), then the line in `Customer.ts` calling `require('.').User` will not be accessing an export that was not defined yet.

Having said that, I think this is not very likely because we are not sorting index files now, thus I am ok to fix this later (or never).

Reply:
Ok. I will leave it out of the scope of this PR, but will work on it.